src/PantherTestCaseTrait.php

```diff
@@ -163,10 +163,12 @@ protected static function createPantherClient(array $options = [], array $kernel
 
         self::startWebServer($options);
 
+        $browserArguments = \array_key_exists('browser_arguments', $options) && \is_array($options['browser_arguments']) ? $options['browser_arguments'] : [];
```

Comment:
I think a possible interpretation of this key might be that you can put a string of options in this like `--some-option --some-other-option`. This of course should not be supported, but I think this mistake can be easily made. Should we raise an exception if this is not an array and just say it must be an array.

With the current implementation, incorrect usage is suppressed. We might be better of forcing correct usage.

Reply:
Do you mean just make sure that `$options['browser_options']` is not a string?

Reply:
I think the best option is to make sure that `browser_arguments` is an array so adding this should be sufficient:
```php
if (!is_array($browserArguments )) {
    throw new UnexpectedValueException('Expected key "browser_arguments" to be an array.');
}

Reply:
I think `!\is_array` is what I'm thinking about

Reply:
@nusje2000 how about throwing a warning using

```php
@trigger_error('...', \E_USER_WARNING);
```